src/server/api/controllers/vessel-reports.controller.js

```diff
@@ -3,13 +3,50 @@
 const HttpError = require("../lib/utils/http-error");
 const knex = require("../../config/db");
 
-//Get vessels reports by vessel id
-const getVesselsReportByVoyageId = async voyage_id => {
-  const vesselReports = await knex.from("vessel_reports").where({ voyage_id });
-  if (vesselReports.length === 0) {
-    return `No voyage exist with that id ${voyage_id}.`;
+//Get vessels reports by voyage_id with limit, offset=0 and order =desc
+const getVesselsReportByVoyageIdWithParams = async (voyage_id, query) => {
```

Comment:
```suggestion
const getVesselsReportByVoyageId = async (voyage_id, query) => {
```

Please keep the original name. We are not going to have a `getVesselsReportByVoyageIdWithoutParams` and overtime we will be adding pagination to most of the endpoints anyway, so not much point in changing the name.

Reply:
changed the method name and added method  swagger.json. Tested.